tag_dedupe/dedupe.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python
+
+import sys
+import subprocess
+import os
+import argparse
+
+AUTH_FILE_PATH_LOCAL = "/auth.json"
+
+def dedupe(input_tag, existing_tags):
```

Comment:
What do you think about "check_if_tag_exists" for this function and the name of the file? It's a little more clear about what it does.